src/website/src/app/documentation/demos/datagrid/selection/selection.html

```diff
@@ -71,7 +71,7 @@ <h2>Selection</h2>
     For row selection mode add <code class="clr-code">[(clrDgRowSelection)]</code> and
     set it to <code class="clr-code">true</code>.
 </p>
-<clr-datagrid [(clrDgSelected)]="rowSelected" [clrDgRowSelection]="true">
```

Comment:
Out of scope ask: can you put the same deprecation warning above this demo as you see https://deploy-preview-3749--vmware-clarity.netlify.com/documentation/datagrid/selection-single?